galsim/phase_psf.py

```diff
@@ -1338,8 +1338,9 @@ def __init__(self, lam_over_diam=None, lam=None, diam=None, tip=0., tilt=0., def
         if max_size is not None:
             from .deprecated import depr
             depr('max_size', 1.4, '',
-                 "The max_size keyword has been removed.  If you have a need for it, "
-                 "please open an issue requesting the functionality.")
+                 "The max_size keyword has been removed.  In its place, the pad_factor keyword can"
+                 "be used to adjust the size of the internal InterpolatedImage.")
```

Comment:
Sure.  I went with doubly commenting to indicate a deprecated comment (description of max_size):

```
        # The original GREAT3 runs used a keyword max_size to improve the efficiency of OpticalPSF.
        # However, we now believe that this keyword introduces additional aliasing and have
        # deprecated its use in GalSim v1.4.  We have retained the original description in the
        # doubly commented block below.

        # # Sometimes the OpticalPSF wants to make a very large image of the PSF.  But if the
        # # PSF image is larger than the postage stamp onto which we will draw the object,
        # # the larger area is mostly wasted.  Setting max_size to the size of the postage stamp
        # # we will eventually draw onto helps avoid wasted calculations.
        # # This is in arcsec, so max_size = stamp_size * pixel_scale
        # # We're using a non-square wcs, so there isn't a well-defined pixel scale.  Use
        # # the maxLinearScale to get the maximum "pixel scale" for the parallelogram.
        # max_size: "$@image.size * wcs.maxLinearScale()"
```

Let me know if you think this is unclear.